ci/code_checks.sh

```diff
@@ -75,6 +75,7 @@ if [[ -z "$CHECK" || "$CHECK" == "doctests" ]]; then
       pandas/core \
       pandas/errors/ \
       pandas/io/ \
+      pandas/plotting/ \
```

Comment:
IIRC this gets us close to just doing everything?

Reply:
Yup this runs doctests on everything worth checking (need to avoid running on `pandas/tests` and `pandas/__init__.py` I think)